rss-reader/Extension/NSDate/NSDate+StringConvertible.m

```diff
@@ -0,0 +1,25 @@
+//
+//  NSDate+StringInitializable.m
+//  rss-reader
+//
+//  Created by Uladzislau on 11/24/20.
+//
+
+#import "NSDate+StringConvertible.h"
+
+@implementation NSDate (StringConvertible)
+
++ (instancetype)dateFromString:(NSString *)string
+                    withFormat:(NSString *)format {
+    NSDateFormatter *formatter = [[NSDateFormatter new] autorelease];
+    formatter.dateFormat = format;
```

Comment:
? SRP violation? You are probably should use category or factory for NSDateFormatter.

Reply:
no

Reply:
Idk, you create, configure and use NSDateFormatter in one method.

Reply:
I can't imagine any case, where I have no need to create and configure NSDateFormatter object to keep a convenience of one-line method